release.md

```diff
@@ -0,0 +1,10 @@
+## Features
```

Comment:
Should this file be here? I am not sure it would have any impact in root, but would there be any reason to keep it?

Reply:
If you want to automate the release message the message needs to be stored somewhere. 

I initially had this file as part of the gitignore, however the pipeline failed because it couldn't reference this file location.

This is the simplest and most crude solution. There are definitely other more elegant solutions we can look at.

Reply:
Ideas for other solutions (In order of difficulty)

1. Don't include release notes at all 
2. Make a folder for each release

```bash
release_notes
  \_ v0.0.1.md
 \_ v0.0.2.md
```

3. Write a script to scrape the changelog and look for the most recent header (A bit tricky, but theoretically doable)